cross/ffmpeg/Makefile

```diff
@@ -1,16 +1,53 @@
 PKG_NAME = ffmpeg
-PKG_VERS = 2.6
-PKG_EXT = tar.gz
+PKG_VERS = 3.0.2
+PKG_EXT = tar.bz2
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = http://www.ffmpeg.org/releases/
+PKG_DIST_SITE = http://www.ffmpeg.org/releases
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/x264 cross/lame
+DEPENDS = cross/libsoxr cross/fdk-aac cross/flac cross/freetype cross/lame cross/libass cross/libbluray
+DEPENDS += cross/libopus cross/libtheora cross/libvorbis cross/libvpx cross/openjpeg cross/openssl
+DEPENDS += cross/rtmpdump cross/speex cross/x264 cross/x265 cross/opencore-amr cross/vo-amrwbenc
 
 HOMEPAGE = http://www.ffmpeg.org/
 COMMENT  = FFmpeg is a complete, cross-platform solution to record, convert and stream audio and video. It includes libavcodec - the leading audio/video codec library
-LICENSE  =
+LICENSE  = LGPL 2.1
 
-CONFIGURE_ARGS = --disable-yasm --enable-gpl --enable-libx264 --enable-libmp3lame --enable-nonfree --enable-cross-compile --arch=$(ARCH) --target-os=linux --cross-prefix=$(TC_PATH)$(TC_PREFIX) --prefix=$(INSTALL_PREFIX) --extra-cflags="-I$(WORK_DIR)/install$(INSTALL_PREFIX)/include" --extra-ldflags="-L$(WORK_DIR)/install$(INSTALL_PREFIX)/lib" --extra-libs=-ldl
+CONFIGURE_ARGS = --target-os=linux --cross-prefix=$(TC_PATH)$(TC_PREFIX) --prefix=$(INSTALL_PREFIX) --extra-cflags="-I$(WORK_DIR)/install$(INSTALL_PREFIX)/include"
```

Comment:
- There are duplicate entries in here (e.g. --enable-gpl)
- Reorder and keep the limited line length, e.g. the `--enable-libsoxr` should be where it's at right now.